README.md

```diff
@@ -386,26 +386,27 @@ kind: KeptnEvaluationDefinition
 metadata:
   name: my-prometheus-evaluation
 spec:
-  source: prometheus
   objectives:
-    - name: query-1
-      query: "xxxx"
+    - keptnMetricRef:
+        name: cpu
+        namespace: ns
       evaluationTarget: <20
-    - name: query-2
-      query: "yyyy"
+    - keptnMetricRef:
+        name: memory
+        namespace: ns
       evaluationTarget: >4
 ```
 
-### Keptn Evaluation Provider
+### Keptn Metrics Provider
```

Comment:
```suggestion
### KeptnMetricsProvider
```
I think we should stick to the "technical" name here and be consistent.  Do you agree?

Reply:
we have it separated in the whole file, so I rather be consistent here, or should we fix it overall ?

Reply:
What do you mean by the "whole file"?  Definitely agree about consistency.  It looks to me that the value of the `kind` field/parameter in the yaml file is `KeptnMetricsProvider` so I was thinking we should use that as a "proper" name everywhere.

Reply:
the whole readme, ok I will fix it

Reply:
Maybe we should discuss it as a team and agree about what we want to do?